.github/workflows/helm-test-workflow.yml

```diff
@@ -88,6 +88,7 @@ jobs:
         if: github.ref == 'ref/head/main'
```

Comment:
are we planning to change head to heads later ?

Reply:
will be taken up as part of tech debt item